requirements.txt

```diff
@@ -1,7 +1,8 @@
-django>=1.11,<1.12
+django>=1.11,<2.1
 djangorestframework>=3.3,<3.8
 django-model-utils
 netdiff>=0.5.1,<=6.0
 openwisp-utils>=0.1.2,<0.2
 jsonfield
 six
+responses
```

Comment:
@phyxkal This is already included in requirements-test.txt

Reply:
You can remove the `responses` from here I suppose

Reply:
owh, okay